buildSrc/build.gradle

```diff
@@ -26,10 +26,11 @@ repositories {
 }
 
 apply plugin: 'kotlin'
+apply plugin: "java-gradle-plugin"
 
 sourceSets {
-    main.java.srcDir 'src'
-    test.java.srcDir 'tst'
+    main.kotlin.srcDir 'src'
```

Comment:
What does this actually change versus the old version?

Reply:
Nothing, just felt more clear since all of this code is Kotlin and not java.